ComplementaryData/modelCuration/modMetsandSmatrix.tsv

```diff
@@ -0,0 +1,788 @@
+s_3932[er]	C50H66N4O29	C50H82N4O37	0	[]	MNXM9328	r_4271	MNXR109973	H-1R1	s_3932[er]: changed C50H66N4O29 to C50H82N4O37 to balance equation	| based on other metFormulas in r_4271 to determine glycan G00012 formula	[]	[]	[]
+s_3943[p]	C10H14N5O8P	C10H12N5O8P	-2	[]	MNXM2497	r_4279	MNXR113234	H4	s_3943[p]: changed C10H14N5O8P to C10H12N5O8P to balance equation	[]	[]	[]	[]
+s_3942[p]	C10H16N5O14P3	C10H12N5O14P3	-4	[]	MNXM4425	r_4279; r_4253; r_4320; r_4321	MNXR113234	H4	s_3942[p]: changed C10H16N5O14P3 to C10H12N5O14P3 to balance equation	[]	[]	[]	[]
+s_0412[c]	C6H14NO8P	C6H13NO8P	0	-1	MNXM162238	r_4286	MNXR100613	H-2	s_0412[c]: changed metFormula C6H14NO8P to C6H13NO8P to balance equation | changed metCharges 0 to -1 to balance equation	[]	[]	[]	[]
+s_3953[c]	C6H13NO5	C6H14NO5	0	1	MNXM533	r_4286	MNXR100613	H-2	s_3953[c]: changed metFormula C6H13NO5 to C6H14NO5 to balance equation | changed metCharges 0 to 1 to balance equation	[]	[]	[]	[]
+s_3964[c]	C7H16NO2	C7H15NO2	1	0	MNXM626	r_4297	MNXR108074	H-1	s_3964[c]: changed metFormula C7H16NO2 to C7H15NO2 to balance equation | changed metCharges 1 to 0 to balance equation	| based on metFormula in MNXM626	[]	[]	[]
+s_3968[c]	C5H6N2O2	C5H5N2O2	0	-1	MNXM1330	r_4299	MNXR95745	H-1	s_3968[c]: changed metFormula C5H6N2O2 to C5H5N2O2 to balance equation | changed metCharges 0 to -1 to balance equation	| based on metFormula in MNXM1330	[]	[]	[]
+s_3972[c]	C10H9NO3	C10H8NO3	0	-1	MNXM1961	r_4301	MNXR109205	H-1	s_3972[c]: changed metFormula C10H9NO3 to C10H8NO3 to balance equation | changed metCharges 0 to -1 to balance equation	| based on metFormula in MNXM1961	[]	[]	[]
+s_3980[c]	C2H3ClO2	C2H2ClO2	0	-1	MNXM1468	r_4305	MNXR109457	H-1	s_3980[c]: changed metFormula C2H3ClO2 to C2H2ClO2 to balance equation | changed metCharges 0 to -1 to balance equation	| based on metFormula in MNXM1468	[]	[]	[]
+s_3982[c]	C10H14O2	C10H13O2	0	-1	MNXM2414	r_4306	MNXR110183	H-1	s_3982[c]: changed metFormula C10H14O2 to C10H13O2 to balance equation | changed metCharges 0 to -1 to balance equation	| based on metFormula in MNXM2414	[]	[]	[]
+s_3984[c]	C15H24O2	C15H23O2	0	-1	MNXM3989	r_4307	MNXR111724	H-1	s_3984[c]: changed metFormula C15H24O2 to C15H23O2 to balance equation | changed metCharges 0 to -1 to balance equation	| based on metFormula in MNXM3989	[]	[]	[]
+s_4012[m]	CHNS	CNS	0	-1	MNXM762	r_4326	MNXR138187	H-1	s_4012[m]: changed metFormula CHNS to CNS to balance equation | changed metCharges 0 to -1 to balance equation	| based on metFormula in MNXM762	[]	[]	[]
+%													
+#for modification of H+/H2O/other met coefficient to balance equation	"#Note: some modifications of H+/H2O may not directly lead to elemental balance e.g. model.S(601, 63), as there may be changes in metFormula for the same reaction"												
+#model.S	model.mets	model.rxns	current_coef	new_coef	elementdiff	Notes	refNotes						
+"601, 56"	s_0794[c]	r_0073	2	1	H-1	"model.S(601, 56): change 2 to 1 to balance equation"	[]						
```

Comment:
It is a little bit risky to use the number as index. try to use rxn id and met id to search the model.S index

Reply:
Issue has been addressed in commit [53c81d9](https://github.com/SysBioChalmers/yeast-GEM/pull/222/commits/53c81d9e0f79b71a8cdac8834826fa0ead86ea5c)